adaptive_breakpoints/test/adaptive_breakpoints_test.dart

```diff
@@ -285,3 +285,190 @@ void main() {
     expect((box as Container).color, trueColor);
   });
 }
+
+/// [AdaptiveContainer] lets you create a [Container] with adaptive constraints.
+///
+/// The AdaptiveContainer does everything a normal container does but with
+/// adaptive constraints. For more information go to one of the links below.
+///
+/// https://api.flutter.dev/flutter/widgets/Container-class.html
+/// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+///
+/// This class is useful whenever you want a container to only be active in
+/// certain [AdaptiveWindowType].
+class _AdaptiveContainer extends StatelessWidget {
```

Comment:
Hmm duplicating this container a 3rd time seems bad. Maybe you could adjust the tests for breakpoints to not require this?